pkg/onchain/website/handler.go

```diff
@@ -52,13 +52,37 @@ func RedirectToDefaultResourceInterceptor(req *interceptor.Interceptor) *interce
 		return nil
 	}
 
-	prefixes := []string{"/browse/", "/thyra/"}
+	// redirect / to /home/index.html
+	if req.Request.URL.Path == "/" {
+		http.Redirect(
+			req.Writer,
+			req.Request,
+			"/home/index.html",
+			http.StatusSeeOther,
+		)
 
-	for _, prefix := range prefixes {
-		if !strings.HasPrefix(req.Request.URL.Path, prefix) {
-			continue
+		return nil
+	}
+
+	redirectPaths := []string{"/home", "/search", "/websiteUploader", "/store"}
+
+	for _, path := range redirectPaths {
+		if req.Request.URL.Path == path || req.Request.URL.Path == path+"/" {
+			http.Redirect(
+				req.Writer,
+				req.Request,
+				path+"/index.html",
+				http.StatusSeeOther,
+			)
+
+			return nil
 		}
+	}
 
+	prefix := "/browse/"
+	if strings.HasPrefix(req.Request.URL.Path, prefix) {
+		// The len(prefix) is used to extract a substring from the req.Request.URL.Path starting from the end of the prefix
+		// string. This is done to remove the prefix from the URL path before splitting it.
 		splited := removeEmptyStrings(strings.Split(req.Request.URL.Path[len(prefix):], "/"))
```

Comment:
this is really ugly :/ can you change it to use a regex?
somthing like
```
regex := regexp.MustCompile(`^/browse/[^/]+`)
if regex.MatchString(path) {
```

Reply:
I remove this peace of code because we remove the /browse endpoint (see comment thread above).